cfg/builder/builder_walk.cc

```diff
@@ -583,9 +608,9 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::Expression *what, BasicBlock
                     InlinedVector<cfg::LocalRef, 2> args{};
                     InlinedVector<core::LocOffsets, 2> argLocs{};
                     auto isPrivateOk = false;
-                    synthesizeExpr(
-                        current, retryTemp, core::LocOffsets::none(),
-                        make_unique<Send>(magic, core::Names::retry(), what->loc, args, argLocs, isPrivateOk));
+                    synthesizeExpr(current, retryTemp, core::LocOffsets::none(),
+                                   make_unique<Send>(magic, core::Names::retry(), what->loc, args.size(), args, argLocs,
```

Comment:
It might be better to make the `numPosArgs` explicitly 0 here.

Reply:
fine either way